### ISSUE TITLE
Add light forem logo for darker themes

### ISSUE THREAD

**Issue by abenettt** [forem/forem#9781]
**Is your feature request related to a problem? Please describe.**
When using a dark theme, as displayed below, the logo on https://forem.dev is barely discernable from the background.

**Describe the solution you'd like**
Maybe use a different logo that is lighter for themes that use dark colors primarily. 

**Describe alternatives you've considered**
Don't do anything or redisign the logo such that it can be used in light and dark environments. 

**Additional context**
![image](https://user-images.githubusercontent.com/7523903/90243824-df55e900-de2f-11ea-8795-bac086956abf.png)

**Comment by krishnadevz** [comment]
> 
> 
> **Is your feature request related to a problem? Please describe.**
> When using a dark theme, as displayed below, the logo on https://forem.dev is barely discernable from the background.
> 
> **Describe the solution you'd like**
> Maybe use a different logo that is lighter for themes that use dark colors primarily.
> 
> **Describe alternatives you've considered**
> Don't do anything or redesign the logo such that it can be used in light and dark environments.
> 
> **Additional context**
> ![image](https://user-images.githubusercontent.com/7523903/90243824-df55e900-de2f-11ea-8795-bac086956abf.png)

Yes logo is not visible and in the dark theme after hovering icons color is appearing is also very dark means there is no use of hover effect and I can change that logo color to white but I am not able to see the content of [homePage.html.erb](https://github.com/forem/forem/blob/master/app/views/homepage/index.html.erb) for editing that. any suggestions/thoughts on this issue @joshpuetz @benhalpern @rhymes @jacobherrington

**Comment by rhymes** [comment]
I maybe wrong as my CSS skills aren't super strong but if the logo is a SVG, can it be applied an [invert filter via CSS](https://developer.mozilla.org/en-US/docs/Web/CSS/filter-function/invert) if the dark theme is selected? Is that

**Comment by krishnadevz** [comment]
> 
> 
> I maybe wrong as my CSS skills aren't super strong but if the logo is a SVG, can it be applied an [invert filter via CSS](https://developer.mozilla.org/en-US/docs/Web/CSS/filter-function/invert) if the dark theme is selected? Is that

we can apply filter like this also means fill color 
![image](https://user-images.githubusercontent.com/42638797/94782430-d2de1d80-03e8-11eb-8543-8760e989f0ab.png)
we can change the fill color to #ffffff and then the result will look like this 
![image](https://user-images.githubusercontent.com/42638797/94782551-07ea7000-03e9-11eb-898d-a5ce85de70df.png)
that will be the good option or else we can do one thing like 
add brightness property to that like 
  --theme-social-icon-invert:  invert(50%) brightness(1.75);
can I make PR for this 
@rhymes @joshpuetz @benhalpern @vaidehijoshi

**Comment by rhymes** [comment]
@krishnadevz ahha please don't tag random DEV members unless there's a specific reason in the conversation :) 

I'm going to ask our design team to provide feedback

**Comment by ludwiczakpawel** [comment]
> add brightness property to that like
> --theme-social-icon-invert: invert(50%) brightness(1.75);

We're going away from using these kind of filters since they aren't bulletproof. It will likely work in that specific scenario but it won't work in many other cases.

The real solution here is letting community admins provide two versions of the logo: one for light themes and one for dark themes. Otherwise we can't guarantee that their logo will look good no matter what theme.

**Comment by mikeydorje** [comment]
Closing as stale. This issue was opened in August 2020. Based on code review:

- The design team's October 2020 recommendation was to let admins provide theme-appropriate logos
- The codebase attempted an automatic `dark_nav_image` approach (added 2019, removed in 2023 migration)
- Current implementation provides admin-configurable logos via Settings::General
- Dark theme CSS applies filters to improve icon visibility (`filter: invert(95%)`)
- Subforems can configure their own logos independently

The underlying issue of logo visibility in dark themes appears to be addressed through the evolution of the theme system and admin configuration options. No activity for 5+ years.

If logo visibility in dark themes remains a problem, please open a new issue with specific reproduction details, screenshots, and the theme configuration you're using.
